js/ui/components/clubhouse.js

```diff
@@ -367,6 +367,9 @@ class ClubhouseNotificationBanner extends MessageTray.NotificationBanner {
 
         button.set_x_expand(false);
         super.addButton(button, callback);
+
+        // @todo: This is a workaround for the missing CSS selector, :nth-child
```

Comment:
nitpick: What's the upstream policy on whether `:nth-child` is likely to be implemented? If it is, then it would be convenient to add a link to the feature request in this comment.

Reply:
If I remember correclty @GeorgesStavracas said "very unlikely", but I think is a good idea to open a conversation upstream in any case.